apps/wing/src/commands/compile.ts

```diff
@@ -1,14 +1,14 @@
 import * as vm from "vm";
 
-import { mkdir, readFile, mkdtemp, rename, rm } from "fs/promises";
+import { readFile, rmSync, mkdirp, move, mkdirpSync, copySync } from "fs-extra";
```

Comment:
Do we really need this extra dependency (pun intended)? All of these are available in `fs` in the node versions we use.

Reply:
`move` is not. It's similar to rename, but handles additional cases. `copy` also isn't (it's `copyFile` in `fs`), it also handles additional cases more gracefully.

Feel free to remove the dep if you'd like.